src/Integration.Vsix.UnitTests/packages.lock.json

```diff
@@ -526,43 +484,42 @@
       },
       "Microsoft.VisualStudio.GraphModel": {
         "type": "Transitive",
-        "resolved": "17.3.32804.24",
```

Comment:
None of the other package lock files referenced this version of the GraphModel, but they do reference version `17.0.31902.203`. We'll have to see if this change makes the `Integration.Vsix.UnitTests` `package.lock.json` more stable going forward.